docs/keto/reference/ory-permission-language.mdx

```diff
@@ -0,0 +1,324 @@
+---
+title: The "Ory Permission Language" Specification
+---
```

Comment:
Can you please use the same frontmatter here as in the other document? 
Please remember to sentence case everything aside from proper names (Ory Permission Language)